go.mod

```diff
@@ -142,12 +143,13 @@ require (
 	go.starlark.net v0.0.0-20200901195727-6e684ef5eeee
 	golang.org/x/exp v0.0.0-20200224162631-6cc2880d07d6 // indirect
 	golang.org/x/lint v0.0.0-20200302205851-738671d3881b // indirect
-	golang.org/x/net v0.0.0-20200904194848-62affa334b73
+	golang.org/x/mod v0.4.1 // indirect
```

Comment:
Where does this come from? If it's needed please keep, but just want to make sure we don't add random stuff by accident. :-)

Reply:
I'm not sure why but it is added by `go get -u github.com/tinylib/msgp`.

I tested it again in a blank project and it confirms it is added by it.

```
$ go get -u github.com/tinylib/msgp
go: github.com/tinylib/msgp upgrade => v1.1.5
go: golang.org/x/mod upgrade => v0.4.1
go: golang.org/x/xerrors upgrade => v0.0.0-20200804184101-5ec99f83aff1
go: golang.org/x/tools upgrade => v0.1.0
go: golang.org/x/sys upgrade => v0.0.0-20210124154548-22da62e12c0c
```

And this is the result.

```
require (
        github.com/tinylib/msgp v1.1.5 // indirect
        golang.org/x/mod v0.4.1 // indirect
        golang.org/x/sys v0.0.0-20210124154548-22da62e12c0c // indirect
        golang.org/x/tools v0.1.0 // indirect
)
```

Reply:
Oh. I unintentionally removed this line in the last commit. But it works without any issue. Can we just safely remove this?

Reply:
If you remove it and it still works I guess so. :-) Just keep the setup without that and we will see. You can view this as resolved. Thanks for checking again!